packages/client/test/sync/txpool.spec.ts

```diff
@@ -9,6 +9,8 @@ tape('[TxPool]', async (t) => {
   const common = new Common({ chain: 'mainnet', hardfork: 'london' })
   const config = new Config({ transports: [], loglevel: 'error' })
 
+  const { PeerPool } = await import('../../lib/net/peerpool')
```

Comment:
this can be imported normally at the top of the file (doesn't need `await import` unless modifying with testdouble)

Reply:
Ah, lost oversight, thanks. Will update.